widget/tabcontainer.go

```diff
@@ -463,15 +531,6 @@ func (b *tabButton) Tapped(e *fyne.PointEvent) {
 	b.OnTap()
 }
 
-func (b *tabButton) setText(text string) {
```

Comment:
Please don't rearrange methods unless they are in the wrong order.

Reply:
I have tried to change as little as possible on the existing code, but is there a description of what is correct order?
Anyway, I have moved setText() back to where if was.

Reply:
I don't think we have it written up the wiki yet, the information is in issue #827